server/src/main/kotlin/ir/armor/tachidesk/server/ServerSetup.kt

```diff
@@ -37,11 +34,9 @@ class ApplicationDirs(
 
 val serverConfig: ServerConfig by lazy { GlobalConfigManager.module() }
 
-val systemTray by lazy { systemTray() }
-
-val androidCompat by lazy { AndroidCompat() }
-
```

Comment:
won't these two get garbage collected later?

specially systemTray that doesn't have other references...

Reply:
Hmm didnt think of that, ill revert it for now, it just seems annoying to have lazy variables controlling weather something is actually initialized

Reply:
I don't have a better solution while doing it in a functional manner either